packages/core/saved-objects/core-saved-objects-server-internal/src/saved_objects_service.test.ts

```diff
@@ -267,7 +269,7 @@ describe('SavedObjectsService', () => {
       expect(migratorInstanceMock.runMigrations).not.toHaveBeenCalled();
     });
 
-    it('calls KibanaMigrator with correct version', async () => {
+    it('calls KibanaMigrator with correct version and waitForMigrationCompletion', async () => {
```

Comment:
We are not really validating the `waitForMigrationCompletion` property in this test, are we?